src/gpu/nvidia/cudnn_softmax.cpp

```diff
@@ -36,7 +37,24 @@ status_t cudnn_softmax_fwd_t::execute(const exec_ctx_t &ctx) const {
         auto arg_src = CTX_IN_SYCL_MEMORY(DNNL_ARG_SRC);
         auto arg_dst = CTX_OUT_SYCL_MEMORY(DNNL_ARG_DST);
 
+        auto src_scales
+                = CTX_IN_SYCL_MEMORY(DNNL_ARG_ATTR_SCALES | DNNL_ARG_SRC);
+        auto dst_scales
+                = CTX_IN_SYCL_MEMORY(DNNL_ARG_ATTR_SCALES | DNNL_ARG_DST);
+
         compat::host_task(cgh, [=](const compat::interop_handle &ih) {
+            if (!pd()->attr()->scales_.defined()) {
+                if (src_scales.get_native_pointer(ih) != NULL) {
+                    host_scales_[0] = reinterpret_cast<float *>(
+                            src_scales.get_native_pointer(ih))[0];
+                }
+                if (dst_scales.get_native_pointer(ih) != NULL) {
+                    host_scales_[1] = reinterpret_cast<float *>(
+                            dst_scales.get_native_pointer(ih))[0];
+                }
+                host_scales_[2] = host_scales_[0] * 1.0f / host_scales_[1];
```

Comment:
`src_scales.get_native_pointer(ih)` and `dst_scales.get_native_pointer(ih)` returns a device memory therefore you cannot work with it as with a host memory. You have to copy them on host using a CUDA copy routine.

Reply:
Currently we added memory copy function in the latest source code.